packages/tsconfig.settings.json

```diff
@@ -11,6 +11,7 @@
     "esModuleInterop": true,
     "moduleResolution": "node",
     "declaration": true,
+    "declarationMap": true,
```

Comment:
I don't think PRs should modify tooling config without an explanation. I started a conversation on slack to normalize our settings across projects.

Having said that, this change is correct, this should have always been `true`.

Reply:
Agreed. I think I was looking at the hardhat repo actually when i fixed this.